react-native-libraries.json

```diff
@@ -13443,9 +13443,9 @@
     "githubUrl": "https://github.com/patlux/react-native-bluetooth-state-manager",
     "ios": true,
     "android": true,
-    "unmaintained": true,
+    "unmaintained": false,
     "alternatives": ["react-native-ble-plx"],
```

Comment:
```suggestion
```

Let's also remove false flaga and `alternatives` array, since those are a hint for unmaintained libraries.